packages/backend-for-frontend/test/api/catalogRouter/getCatalog.test.ts

```diff
@@ -88,9 +88,13 @@ describe("API GET /catalog", () => {
     { query: { offset: 0, limit: 201 } },
     { query: { offset: "invalid", limit: 10 } },
     { query: { offset: 0, limit: "invalid" } },
-  ])("Should return 400 if passed an invalid parameter", async ({ query }) => {
-    const token = generateToken(authRole.ADMIN_ROLE);
-    const res = await makeRequest(token, query as typeof defaultQuery);
-    expect(res.status).toBe(400);
-  });
+    { query: { personalData: "invalid" } },
```

Comment:
My bad. This should be like this, otherwise it returns 400 for a different reason (limit and offset are required). Same thing for the `getCatalogEServiceTemplates`.
```suggestion
    { query: { ...defaultQuery, personalData: "invalid" } },
```